modules/gitops-configuring-respect-rbac-using-gitops.adoc

```diff
@@ -0,0 +1,100 @@
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: example-argocd #<1>
+spec:
+  controller:
+    respectRBAC: strict #<2>
+----
+<1> Specify the name of the Argo CD instance.
+<2> Specify the value of the `.spec.controller.respectRBAC` key in the Argo CD resource. You can set it to `normal` or `strict`. If the value is set to `strict`, Argo CD performs additional validations of RBAC resources to determine permissions. This setting increases the number of API calls to the server as it is more accurate compared to `normal`. Switch to `strict` if Argo CD reports errors indicating it cannot access resources when the value is set to `normal`. If the value is set to `normal`, Argo CD determines its RBAC permissions by attempting to list each resource. By default it is recommended to start with `normal`, as it balances accuracy and speed because resource listing is a lightweight operation.
+
+. Apply the changes to the YAML object file by running the following command.
++
+[source,terminal]
+----
+$ oc apply -f argocd-resource.yaml -n argo-cd-instance #<1>
+----
+<1> Specify the name of the YAML file that includes the Argo CD resource and the namespace that hosts the Argo CD instance.
++
+. Verify the successful installation of Argo CD by checking the status of the ArgoCD `.status.phase` field. The {gitops-shortname} Operator sets this field to `Available` when the installation is complete. Run the following command to check the status:
++
+[source,terminal]
+----
+oc get argocd <argocd-instance-name> -n <argocd namespace> -o jsonpath='{.status.phase}' #<1>
+----
+<1> Replace `<argocd-instance-name>` with the name of your Argo CD instance for example, `example-argocd`.
+
+. After the Argo CD instance is created, verify that the `resource.respectRBAC` parameter in the config map resource is updated successfully by completing the following steps.
+.. To retrieve the contents of the `argocd-cm` config map, run the following command:
++
+[source,terminal]
+----
+oc get cm argocd-cm -n <argocd namespace> -o yaml
+----
+.. Verify that the `argocd-cm` config map contains the `resource.respectRBAC` parameter.
+
+[id="configuring-respectRBAC-using-the-web-UI_{context}"]
+== Configuring respectRBAC using the Web UI
+
+You can configure `respectRBAC` in the Web UI.
+
+.Procedure
+
+. Log in to the {OCP} web console. 
+
+. In the *Administrator* perspective of the web console, click *Operators* -> *Installed Operators*.
+
+. Create or select the project where you want to install the user-defined Argo CD instance from the *Project* list.
+
+. Select *{gitops-title}* from the installed Operators list and click the *Argo CD* tab.
+
+. Configure the `respectRBAC` parameter in the *Argo CD* tab.
++
+[source,yaml]
+----
+spec:
+  controller:
+    respectRBAC: strict
+----
+
+. Click *Create*.
++
+After successful installation, the Argo CD instance is listed under *Argo CD* tab and its Status should be set to *Available*.
```

Comment:
```suggestion
After successful installation, verify that the Argo CD instance is listed under the *Argo CD* tab and the *Status* is *Available*.
```

- Since it's console, wondering if Status should be in bold ; if not please use back ticks.
- Avoid using should
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s

> Whenever possible, rewrite in a more direct way. Do not use should to refer to an action that must be performed. For example, when parameters must be reset, write “Reset the parameters” and not “You should reset the parameters”.

Reply:
Fixed.